src/infill.h

```diff
@@ -87,6 +89,7 @@ class Infill
     , zag_skip_count(zag_skip_count)
     , apply_pockets_alternatingly(apply_pockets_alternatingly)
     , pocket_size(pocket_size)
+    , maximum_resolution(maximum_resolution)
```

Comment:
In infill the "maximum_resolution" is used only fro creating ZigzagConnectorProcessor, maybe it is better to define this variable in ZigzagConnectorProcessor class by default, for now it will help to prevent the number update from different places.